### PR TITLE
Misc logging improvements

### PR DESCRIPTION
**Pull request by paulgb**
This makes a bunch of ergonomic fixes to logs; comments inline.

### CODE REVIEW

File: plane/src/controller/mod.rs [hunk 1]
```diff
@@ -158,8 +158,9 @@ impl ControllerServer {
 
         let trace_layer = TraceLayer::new_for_http()
             .make_span_with(DefaultMakeSpan::new().level(Level::INFO))
-            .on_request(DefaultOnRequest::new().level(Level::INFO))
-            .on_response(DefaultOnResponse::new().level(Level::INFO));
+            .on_request(DefaultOnRequest::new().level(Level::DEBUG))
+            .on_failure(DefaultOnFailure::new().level(Level::WARN))
+            .on_response(DefaultOnResponse::new().level(Level::DEBUG));
```
Comment (paulgb) [inline plane/src/controller/mod.rs:163]: logging each request is too chatty, only log failures.

File: plane/src/drone/docker/commands.rs [hunk 2]
```diff
@@ -43,7 +43,7 @@ pub async fn pull_image(
     while let Some(next) = result.next().await {
         let info = next?;
         if let Some(progress) = info.progress_detail {
-            tracing::info!(?progress, "Image pull progress.");
+            tracing::debug!(?progress, "Image pull progress.");
```
Comment (paulgb) [inline plane/src/drone/docker/commands.rs:46]: these are very chatty (log every progress tick)

File: plane/src/drone/docker/mod.rs [hunk 3]
```diff
@@ -116,12 +117,20 @@ impl PlaneDocker {
...
+                        "Ignoring event with invalid backend ID."
+                    );
                     return None;
                 }
             };
 
-            tracing::info!(exit_code, "Received exit code");
+            tracing::info!(
+                exit_code,
+                backend_id = backend_id.as_value(),
```
Comment (paulgb) [inline plane/src/drone/docker/mod.rs:131]: log backend id here

File: plane/src/drone/docker/mod.rs [hunk 4]
```diff
@@ -235,9 +244,9 @@ impl PlaneDocker {
             {
                 Ok(result) => {
                     let num_images_deleted = result.images_deleted.map(|d| d.len()).unwrap_or(0);
-                    tracing::info!(num_images_deleted, "Removed dangling images.");
+                    tracing::info!(num_images_deleted, "Pruning images.");
```
Comment (paulgb) [inline plane/src/drone/docker/mod.rs:247]: "dangling" has a special meaning in docker and we don't necessarily abide by it here

File: plane/src/types/backend_state.rs [hunk 5]
```diff
@@ -36,14 +36,32 @@ pub enum BackendStatus {
...
+            BackendStatus::Ready => valuable::Value::String("ready"),
+            BackendStatus::Terminating => valuable::Value::String("terminating"),
+            BackendStatus::Terminated => valuable::Value::String("terminated"),
+        }
+    }
+
+    fn visit(&self, visit: &mut dyn valuable::Visit) {
+        visit.visit_value(self.as_value())
+    }
+}
```
Comment (paulgb) [inline plane/src/types/backend_state.rs:55]: auto-generated implementation of enums by valuable isn't great

File: plane/src/types/backend_state.rs [hunk 6]
```diff
@@ -68,6 +86,97 @@ pub enum BackendState {
     },
 }
 
+impl valuable::Valuable for BackendState {
```
Comment (paulgb) [inline plane/src/types/backend_state.rs:89]: this creates log entries of the form `{"status": "scheduled"}` or `{"status": "waiting", "address": "..."}`

**Review by rolyatmax — APPROVED** [review]

**Pull request merged**